Resources/Storage/SRM2Storage.py

```diff
@@ -899,7 +899,12 @@ def __getFile( self, src_url, dest_file ):
       self.log.debug( "__getFile: Local file already exists %s. Removing..." % dest_file )
       os.remove( dest_file )
     srctype = self.defaulttype
-    src_spacetokendesc = self.spaceToken
+    # lcg-cp download from a bestman2 SE does not work when src_spacetokendesc specified
+    backendType = gConfig.getValue( '/Resources/StorageElements/%s/BackendType' % ( self.name ), '' )
```

Comment:
I am very surprised by this ! 
In any case, instead of calling gConfig, you can directly access the option with ```self.se.options.get('BackendType')```